ios/RNTextInputMask/InputMask/InputMask.xcodeproj/project.pbxproj

```diff
@@ -537,7 +537,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 3.0;
+				SWIFT_VERSION = 4.2;
```

Comment:
I don't know why these are showing up as changed again, they're definitely 4.2 in the actual file